spec/models/embed/envelope_spec.rb

```diff
@@ -0,0 +1,35 @@
+require "rails_helper"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.